src/Microsoft.ML/Microsoft.ML.csproj

```diff
@@ -1,44 +1,24 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$(RepoRoot)eng/pkg/Pack.props"/>
+<Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Nit: We try to not use netstandard2.0 as the TFM if not necessary for compilation. I would update this to NetCurrent or whatever property exists in this repo to indicate the latest supported .NETCoreApp TFM.

Reply:
This package *bundles* a set of libs that are all built for `netstandard2.0` so it needs a `netstandard2.0` TFM.  Here the TFM is important as it's what's used when resolving the P2Ps and what folder all their output is placed in.

Reply:
Clarified with comment.